### PR TITLE
The description of /api​/clusters_mgmt​/v1​/aws_inquiries​/vpcs is not correct

### PR DESCRIPTION
**Pull request by olisakov**
https://issues.redhat.com/browse/SDA-5182

### CODE REVIEW

File: model/clusters_mgmt/v1/key_rings_inquiry_resource.model [hunk 1]
```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 // Manages the collection of key rings inquiry.
 resource KeyRingsInquiry {
-	// Retrieves the list of available regions of the cloud provider.
+	// Retrieves the list of available keys of the cloud provider.
```
Comment (tzvatot) [inline model/clusters_mgmt/v1/key_rings_inquiry_resource.model:19]: Key rings

File: model/clusters_mgmt/v1/key_rings_inquiry_resource.model [hunk 2]
```diff
@@ -27,12 +27,12 @@ resource KeyRingsInquiry {
 
 		// Number of items that will be contained in the returned page. As this collection
 		// doesn't support paging or searching the result will always be the total number of
-		// regions of the provider.
+		// keys of the provider.
```
Comment (tzvatot) [inline model/clusters_mgmt/v1/key_rings_inquiry_resource.model:30]: key rings

File: model/clusters_mgmt/v1/key_rings_inquiry_resource.model [hunk 3]
```diff
@@ -27,12 +27,12 @@ resource KeyRingsInquiry {
...
 		in out Size Integer = 100
 
 		// Total number of items of the collection that match the search criteria,
 		// regardless of the size of the page. As this collection doesn't support paging or
-		// searching the result will always be the total number of available regions of the provider.
+		// searching the result will always be the total number of available keys of the provider.
```
Comment (tzvatot) [inline model/clusters_mgmt/v1/key_rings_inquiry_resource.model:35]: key rings

File: model/clusters_mgmt/v1/key_rings_inquiry_resource.model (hunk 1 above)
Comment (zvikorn) [inline model/clusters_mgmt/v1/key_rings_inquiry_resource.model:19]: key rings

File: model/clusters_mgmt/v1/key_rings_inquiry_resource.model (hunk 2 above)
Comment (zvikorn) [inline model/clusters_mgmt/v1/key_rings_inquiry_resource.model:30]: same

File: model/clusters_mgmt/v1/key_rings_inquiry_resource.model [hunk 4]
```diff
@@ -27,12 +27,12 @@ resource KeyRingsInquiry {
...
 		out Total Integer
```
Comment (zvikorn) [inline model/clusters_mgmt/v1/key_rings_inquiry_resource.model:36]: same

File: model/clusters_mgmt/v1/vpcs_inquiry_resource.model [hunk 5]
```diff
@@ -16,9 +16,9 @@ limitations under the License.
 
 // Manages the collection of vpcs inquiry.
 resource VpcsInquiry {
-	// Retrieves the list of available regions of the cloud provider.
+	// Retrieves the list of available vpcs of the cloud provider pair specific region.
```
Comment (zvikorn) [inline model/clusters_mgmt/v1/vpcs_inquiry_resource.model:19]: for specific region

**Comment by olisakov** [comment]
@zvikorn @tzvatot  PTAL

**Comment by zvikorn** [comment]
LGTM.
However, you need a maintainer to approve it.

**Review by jhernand — APPROVED** [review]
Looks good, thanks @olisakov.

**Pull request merged**